### ISSUE TITLE
it's good tool ,can this support Oracle  DB ?

### ISSUE THREAD

**Issue by roger68** [tokern/lineage#8]

**Comment by vrajat** [comment]
Yes. It can work with Oracle. I have to make sure Oracle queries parse correctly. Ar you open to sharing some real world queries so that I can test ? 

I don’t have a real workload from Oracle to test with.
